CONTRIBUTING.md

```diff
@@ -33,7 +33,6 @@ When submitting a pull request (PR), please use the following guidelines:
   using "Feature/Change" template. Non-trivial features should include unit tests covering the new functionality. Open
   a "Proposal" issue for large changes.
 - Bugfixes should include a unit test or integration test reproducing the issue.
-- Do not use author tags/information in the code.
```

Comment:
it appears that auto check would only look in java files and not in other files, so maybe it is still good to leave this.

Reply:
+1 for leaving this in the docs

Reply:
@himanshug @clintropolis  Thanks. Addressed your comments, it has rolled back to its previous state.

Reply:
I think nobody ever adds this in non-Java files. It appears when people have default Java class templates in IDEs which insert this stuff. So IMO it's OK to remove this line from this file because it touches on a low-important topic, not in line with other, higher-level items in this list.

Such item may be added to an extended PR checklist though.